packages/niivue/src/nvdocument.ts

```diff
@@ -192,6 +192,7 @@ export type NVConfigOptions = {
   isAlphaClipDark: boolean
   gradientOrder: number
   gradientOpacity: number
+  gradientAmount: number
```

Comment:
I am wondering if this should be something more semantically specific like `illumination`?

Reply:
Sounds good to me.  @neurolabusc are you okay with changing the name to illumination?

Reply:
@cdrake and @thewtex , is this conversation resolved? Is the current naming in this PR the name we are going with?